public/scripts/select.js

```diff
@@ -0,0 +1,12 @@
+
+(function(module) {
+ var select = {}
```

Comment:
This seems more appropriate to be part of a larger piece of code rather than it's own little module.